docs/custom_text_ops.md

```diff
@@ -11,11 +11,11 @@
 |StringRegexReplace| Supported  |
 |StringSplit | Supported       |
 |StringUpper  | Supported     |
-|StringSlice | Under development|
-|StringLength | Under development |
+|StringLength | Supported |
+|StringConcat | Supported |
+|VectorToString| Supported |
 |StringToVector|  Under development|
-|VectorToString| Under development |
-
+|StringSlice | Under development|
```

Comment:
Is this slice same as the one here: https://github.com/onnx/onnx/blob/master/docs/Operators.md#slice